tt_metal/llrt/tlb_config.cpp

```diff
@@ -185,7 +185,7 @@ void configure_static_tlbs(tt::ARCH arch, chip_id_t mmio_device_id, const metal_
     // Setup static TLBs for all worker cores
     for (auto &core : statically_mapped_cores) {
         auto tlb_index = get_static_tlb_index(core);
-        device_driver.configure_tlb(mmio_device_id, core, tlb_index, address);
+        device_driver.configure_tlb(mmio_device_id, core, tlb_index, address, TLB_DATA::Strict);
```

Comment:
can you add a comment about why we need to use strict tlb

Reply:
> can you add a comment about why we need to use strict tlb

done